MapboxSearch/ui/src/main/java/com/mapbox/search/ui/adapter/autofill/AutofillItemsCreator.kt

```diff
@@ -30,7 +30,11 @@ internal class AutofillItemsCreator(
 
         return suggestions.map { suggestion ->
             val locationRequest = locationEngine.lastKnownLocation(context)
-            val distance: Double? = locationRequest.value?.distanceTo(suggestion.coordinate)
+            // TODO: somehow this is required to calculate distances,
+            // would be nice to receive 'distance' in the response instead
```

Comment:
`AddressAutofillSuggestion.coordinate` should be removed. Now we should expose `distanceMeters` field in `AddressAutofillSuggestion` and use it for distance calculation, if it's available for the address autofill backend (check it, please).

Reply:
@DzmitryFomchyn we are not planning to offer distanceMeters as a feature in Autofill until we have a request from a customer. The current thinking is that this is not particularly useful for form autofill.

Reply:
The `coordinate` field was removed.